AzureCalling/Controllers/CallViewController.swift

```diff
@@ -134,9 +135,31 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
     }
 
     private func openAudioDeviceDrawer() {
-        let audioDeviceSelectionViewController = AudioDeviceSelectionViewController()
-        audioDeviceSelectionViewController.modalPresentationStyle = .overCurrentContext
-        present(audioDeviceSelectionViewController, animated: false, completion: nil)
+        let audioDeviceSelectionViewController = AudioDeviceSelectionDataSource()
+        audioDeviceSelectionViewController.createAudioDeviceOptions()
+
+        let bottomDrawerViewController = BottomDrawerViewController(
+            dataSource: audioDeviceSelectionViewController,
+            delegate: audioDeviceSelectionViewController)
+        present(bottomDrawerViewController, animated: false, completion: nil)
+    }
+
+    private func openParticipantListDrawer() {
+        let participantListViewController = ParticipantListDataSource()
+        let participantListInfo = ParticipantListInfo(
+            localDisplayName: callingContext.displayName,
+            localIsMuted: callingContext.callIsMuted,
+            remoteParticipants: callingContext.remoteParticipants.map { (remoteParticipant) -> RemoteParticipantInfo in
```

Comment:
It would be clear if the converting remoteParticipant and ListInfo are separated. 
```let remoteParticipantInfoList = callingContext.remoteParticipants.map .....
let participantListInfo = ParticipantListInfo(
            localDisplayName: callingContext.displayName,
            localIsMuted: callingContext.callIsMuted,
            remoteParticipants:remoteParticipantInfoList